ptx/sec_optimization.ptx

```diff
@@ -1087,7 +1151,21 @@
               that is, <m>S = kwh^2</m> for some constant <m>k</m>.
             </p>
             <image xml:id="img_04_03_ex_13" width="47%">
-              <description></description>
+              mage xml:id="img_04_03_ex_08" width="47%">
+              <shortdescription>
+              A circle with a rectangle divided diagonally inscribed inside it.
+              </shortdescription>
+              <description> 
+                <p>
+               A circular log with diameter of 12 inches has rectangle inscribed inside. The rectangle is divided diagonally forming
+               two equal right-angle triangles. The line that divides it is the diameter
+              </p>
+
+              <p>
+                The shorter side of the rectangle is labelled <m>w</m> which serves as the cross sectional width and the 
```

Comment:
To keep my co-authors/editors happy, I think we need to use the American spelling "labeled".
(I would also double the l otherwise.)